source/System.cpp

```diff
@@ -303,6 +304,155 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 				object.message = &UNINHABITEDPLANET;
 		}
 	}
+	
+	// Calculate efficiency of ramscoops and solar panels.
+	std::list<double> ramscoopModifiers;
+	std::list<double> solarPanelModifiers;
+	
+	// Get list of modifiers associated with each star in the system and remember how many.
+	int starCount = 0;
+	for(StellarObject &object : objects)
+    {
+        if(!object.IsStar())
+            continue;
+
+        string starName = object.GetSprite()->Name();
+        
+        // Add the correct value to the lists. 
```

Comment:
Maybe bring these out into a data file.
```
star a0
	brightness 1.8
	wind 0.8
```
That would allow plugin makers to give their own sprites brightness and wind values.